### ISSUE TITLE
Low to no activity

### ISSUE THREAD

**Issue by imraansarwar** [apache/incubator-predictionio#351]
The activity on the project is getting lower to lowest. Can you shed some light on the future road-map of the project?

Regards,
Imran

**Comment by lucasbm88** [comment]
I do not see this way, perhaps you did not subscribe to the project developers mailing list: dev@predictionio.incubator.apache.org

**Comment by imraansarwar** [comment]
Yes, I have not yet subscribed, however I looked at the "Graphs" and it shows quite less activity. 

PredictionIO seems a great project, and I wish it a great success. I'm evaluating different ML frameworks and found this quite interesting.
Can you point me to any feature roadmap of the project?

**Comment by dszeto** [comment]
Hi, please redirect usage questions to user@predictionio.incubator.apache.org, and development question to dev@predictionio.incubator.apache.org for more exposure to the rest of the community. Subscription instructions can be found at http://predictionio.incubator.apache.org/support/.

We are looking to release 0.11 shortly and post a tentative roadmap for discussion to dev@. Stay tuned!
